README.md

```diff
@@ -155,102 +159,30 @@ need to manually deploy, use the deploy script and the `default` project:
 
 ## Writing for flutter.dev
 
-(Eventually, this section should be expanded to its own page.)
 
-## Syntax highlighting
+The [site-shared](https://github.com/dart-lang/site-shared) repo
+contains infrastructure used by most of our Dart and Flutter websites.
 
-The easiest way to syntax highlight a block of code is to wrap
-it with triple backticks followed by the language.
+For more information on using/writing for this repo,
+refer to the following docs:
 
-Here's an example:
+* [Infrastructure](https://github.com/dart-lang/site-shared/blob/master/doc/infrastructure.md)
+* [Markdown](https://github.com/dart-lang/site-shared/blob/master/doc/markdown.md)
+* [Examples (and code excerpts)](https://github.com/dart-lang/site-shared/blob/master/doc/examples.md)
+* [Code excerpts](https://github.com/dart-lang/site-shared/blob/master/doc/code-excerpts.md)
 
-```dart
-class ExampleWidget extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return Container();
-  }
-}
-```
-
-## (Deprecated) Advanced stylization of code blocks
-
-Do you want to highlight (make the background yellow)
-code inside a code block? Do you want to strike-through
-code inside a code block? We got that!
-
-For syntax highlighting, plus yellow highlighting
-and strike-through formatting, use the `prettify` tag
-with additional custom inline markup.
-
-If you want to highlight a specific bit of code, use the
-`[[highlight]]highlight this text[[/highlight]]` syntax
-with the `prettify` tag.
-
-For example:
-
-    {% prettify dart %}
-    void main() {
-      print([[highlight]]'Hello World'[[/highlight]]);
-    }
-    {% endprettify %}
-
-If you want to strike-through a specific bit of code, use the
-`[[strike]]highlight this text[[/strike]]` syntax
-with the `prettify` tag.
-
-For example:
-
-    {% prettify dart %}
-    void main() {
-      print([[strike]]'Hello World'[[/strike]]);
-    }
-    {% endprettify %}
-
-The `prettify` plugin will also unindent your code.
-
-If you want to see how this functionality was added to this site, refer to
-[this commit](https://github.com/flutter/website/commit/ea15f52fe47d3a7b6313ac58d07c66f3b29fe74d).
+Also check out the site-shared
+[wiki](https://github.com/dart-lang/site-shared/wiki)
```

Comment:
wiki) -> wiki):

Should we say something about moving this stuff to site-shared/doc?
Should we refer to site-shared/doc above?